tests/unit/core/test_Strategy.py

```diff
@@ -0,0 +1,209 @@
+from twisted.trial import unittest
+import random
```

Comment:
randomness in unit tests is generally discouraged.  If you're going to do it, at least set the seed to something known so that the results of each run is predictable.